src/nidcpower/metadata/enums.py

```diff
@@ -93,16 +116,22 @@
         'values': [
             {
                 'name': 'CURRENT_REGULATE',
+                'prefix': None,
                 'value': 13613,
 'documentation': {
-'description': '',
+'description': '''
+
```

Comment:
Is this desirable, to have nothing but an couple of empty new lines in the description?

Reply:
Fixed